MBBSEmu/HostProcess/GlobalRoutines/InstantMessageGlobal.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using MBBSEmu.Extensions;
+using MBBSEmu.Memory;
+using MBBSEmu.Module;
+using MBBSEmu.Session;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using MBBSEmu.Session.Enums;
+
+namespace MBBSEmu.HostProcess.GlobalRoutines
+{
+    public class InstantMessageGlobal : IGlobalRoutine
+    {
+        public bool ProcessCommand(byte[] command, ushort channelNumber, PointerDictionary<SessionBase> sessions, Dictionary<string, MbbsModule> modules)
+        {
+            var commandString = Encoding.ASCII.GetString(command).TrimEnd('\0');
```

Comment:
We can do `if(command.Length < 6) return;` here to bail out quickly before converting the input. Also simplifies the below IF statement.